deployment/cdk/opensearch-service-migration/lib/network-stack.ts

```diff
@@ -257,6 +259,17 @@ export class NetworkStack extends Stack {
                     defaultDeployId: deployId,
                     parameter: MigrationSSMParameter.OS_CLUSTER_ENDPOINT
                 });
+                // This is a somewhat surprsing place for this non-network related set of parameters, but it pairs well with
+                // the OS_CLUSTER_ENDPOINT parameter and is helpful to ensure it happens. This probably isn't a long-term place
+                // for it, but is helpful for the time being.
+                if (props.targetClusterUsername && props.targetClusterPasswordSecretArn) {
+                    createMigrationStringParameter(this,
```

Comment:
This is currently created in opensearch-domain stack and container stack, if including here it needs to be removed from the other two otherwise there will be a deployment conflict from multiple stacks creating the same ssm parameter

Reply:
It's only created here if `targetEndpoint`, which is the condition under which opensearch-domain is not created at all, so I don't think there's a conflict between these two.

Good catch on the os-container one--I've adjusted that to only create the SSM param in the case of demoAdmin, which is when this one won't be used.

This is a bit cobbled together and I think very due for an overhaul/simplification when we start passing in target cluster details more systematically.